OrbitBase/include/OrbitBase/Path.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
```

Comment:
I would suggest renaming the file to something more specific, like `ExecutablePath`. `Path` is way too generic now.

Reply:
Good point. Especially since OrbitCore/Path is still there. A different name might be less confusing.

Reply:
done.